src/card.css

```diff
@@ -0,0 +1,139 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable declaration-block-trailing-semicolon */
+/* stylelint-disable shorthand-property-no-redundant-values */
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+
+.product-card {
+  display: block;
+  background-position: center;
+  height: 406px;
+  width: 198px;
+  margin: 0 auto;
+  border: 1px solid #f3f3f3;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.product-image {
+  position: flex;
+  background-image: url("images/imac.jpeg");
```

Comment:
You've already add it in src attribute, choose only one variand and remove another